config.go

```diff
@@ -131,6 +136,53 @@ func (log Logger) LoadConfiguration(filename string) {
 	}
 }
 
+/* 
+   Replace all instances of `${var}` in the string with the value of the environment variable `var`.
```

Comment:
It looks like this only support ${var}. It'd be nice if we also supported $var, but it's probably not a deal breaker.